benefits/eligibility/context/flow.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass, asdict
+from typing import Optional
+
+from django.utils.translation import gettext_lazy as _
+
+from benefits.core.context import SystemName
+from benefits.routes import routes
+
+
+@dataclass
+class CTAButton:
+    text: str
+    route: str
+    fallback_text: Optional[str] = None
+    extra_classes: Optional[str] = None
+
+
+@dataclass
+class EligibilityStart:
+    page_title: str
+    headline_text: str
+    eligibility_item_template: str
```

Comment:
Maybe we can take a follow-up item to refactor this field away from using these templates, and to encode the item strings directly in context.

Reply:
Yep, great idea! Created https://github.com/cal-itp/benefits/issues/2759 for it

Reply:
I put it in the sprint, but feel free to change that if you disagree